application/microservices/entando-hub-catalog-ms/src/main/java/com/entando/hub/catalog/persistence/entity/Catalog.java

```diff
@@ -0,0 +1,29 @@
+package com.entando.hub.catalog.persistence.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@SequenceGenerator(name = "catalog_id", sequenceName = "SEQ_CATALOG_ID", allocationSize = 1)
+public class Catalog {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "catalog_id")
+    private Long id;
+
+    @Column(unique = true)
+    private Long organisationId;
```

Comment:
Shouldn't this be a `@ManyToOne`?

Reply:
@cecchisandrone since there is only 1 catalog for each organisation, should it be a `@OneToOne`?

Reply:
yes, right